supplements/tomb-of-annihilation/items.xml

```diff
@@ -0,0 +1,183 @@
+            <p class="indent">If you aren't undead, you must make a DC 16 Constitution saving throw each time you use the amulet to teleport. On a failed saving throw, the black skull cackles as you are transformed in transit. The transformation takes effect as soon as you arrive at the destination, and is determined randomly by rolling percentile dice and consulting the Black Skull Transformation table.</p>
+            <table>
+                <thead>
+                    <tr><td>d100</td><td>Transformation</td></tr>
+                </thead>
+                <tr><td>01-20</td><td>The symbol of Acererak is burned into your flesh, a curse that can only be removed with a remove curse spell or similar magic. Until the curse ends, your hit points can't be restored by magic.</td></tr>
+                <tr><td>21-35</td><td>You grow larger as if affected by an enlarge/reduce spell, except the effect lasts for 1 hour.</td></tr>
+                <tr><td>36-50</td><td>You grow smaller as if affected by an enlarge/reduce spell, except the effect lasts for 1 hour.</td></tr>
+                <tr><td>51-70</td><td>You arrive at the destination wearing nothing but the amulet of the black skull. Everything else that you were wearing or carrying appears in a random unoccupied space within 100 feet of you.</td></tr>
+                <tr><td>71-95</td><td>You are paralyzed for l minute or until this effect is ended with a lesser restoration spell or similar magic.</td></tr>
+                <tr><td>96-00</td><td>You become petrified. Th is effect can be ended only with a greater restoration spell or similar magic.</td></tr>
+            </table>
+        </description>
+        <setters>
+            <set name="category">Wondrous Items</set>
+            <set name="weight">0 lb.</set>
+            <set name="type">Wondrous Item</set>
+            <set name="rarity">Very Rare</set>
+            <set name="attunement">true</set>
+        </setters>
+    </element>
+
+<!-- BOOKMARK -->
+    <element name="Bookmark" type="Magic Item" source="Tomb of Annihilation" id="ID_WOTC_TOA_MAGIC_ITEM_BOOKMARK">
+        <description>
+            <p>This +3 dagger belongs to Artus Cimber (see appendix D). While you have the dagger drawn, you can use a bonus action to activate one of the following properties:</p>
+            <p class="indent">Cause a blue gem set into the dagger's pommel to shed bright light in a 20-foot radius and dim light for an additional 20 feet, or make the gem go dark.</p>
+            <p class="indent">Turn the dagger into a compass that, while resting on your palm, points north. </p>
+            <p class="indent">Cast dimension door from the dagger. Once this property is used, it can't be used again until the next dawn .</p>
```

Comment:
```suggestion
            <p class="indent">• Cast dimension door from the dagger. Once this property is used, it can't be used again until the next dawn.</p>
```